snap/snapcraft.yaml

```diff
@@ -237,64 +242,84 @@ apps:
     adapter: none
     after:
       - security-bootstrap-redis
+      # have consul token generated first for this service
+      - security-consul-bootstrapper
       - security-proxy-setup
     command: bin/core-metadata -confdir $SNAP_DATA/config/core-metadata/res -cp -r
     command-chain:
       - bin/security-secret-store-env-var.sh
     environment:
       SECRETSTORE_TOKENFILE: $SNAP_DATA/secrets/edgex-core-metadata/secrets-token.json
+      SERVICE_CONFIGACCESSTOKENFILE: $SNAP_DATA/secrets/edgex-core-metadata/consul-token
```

Comment:
Since we have to override here due to the change from `/tmp` to  `$SNAP_DATA` should we do the same in secure docker and have the default in the config files be set to empty? That way in non-secure mode they don't see the funky `Warning`?

If we did take that approach, that `Warning` could become an `Error`

Reply:
not sure which way is better: that would increase quite a few env. overrides in secure docker file

Reply:
Yea, that is what I was try to avoid, but now that we have to have them in Snap, it would make it consistent across snap and docker. I also felt having it in the default (which is constantly non-secure) is a bit of a code smell. ;-)

Reply:
oh, man, now we would resurrect all your PRs done in last few days? ;-)

Reply:
Yup, I can do ASC and SDK. You can just move forward with these PRs assuming overrides needed.

Reply:
awesome, thanks Lenny!

Reply:
Note:   now will change to clean up these envs and from configuration as it will be no longer needed once https://github.com/edgexfoundry/go-mod-bootstrap/pull/202 is merged in and uses that version of `go-mod-bootstrap`.

Reply:
those file-based configuration of edgex-go services are removed now, so no longer an issue here.